web/gds-user-ui/src/components/Addresses/AddressList.tsx

```diff
@@ -12,7 +12,9 @@ function AddressList() {
 
   const handleAddressClick = () => {
     append({
-      address_type: '',
+      /* Set the default address type value for additional any addresses 
+      to match the default address type provided by backend. */
+      address_type: 'ADDRESS_TYPE_CODE_BIZZ',
```

Comment:
Might be useful to store the default values somewhere so we can import them when needed throughout the code base and there's less inconsistency there.